Add a separate entry.sh file

frankhaussmann opened the pull request:
This is to avoid rendering the file from the Dockerfile as was done
initially. It's just ugly...

Commit:
Add a separate entry.sh file

This is to avoid rendering the file from the Dockerfile as was done
initially. It's just ugly...

dassbj01 commented on Dockerfile:
```diff
@@ -52,19 +52,7 @@ RUN for verb in help \
-
+ADD ./entry.sh /
 VOLUME /acme.sh
 
```
please add the /tls folder as volume explicitly ..

frankhaussmann replied:
Changed this now...

Commit:
Explicitely mount volumes and create mount points

The mount point dirs have not been set up explicitely. They are now
created in the Dockerfile. Also the volumes to be mounted on these mount
points are now put in there to make the intent clear.

Commit:
Fix Dockerfile

tried to create already existing directory in the Dockerfile.
